res/TensorFlowLiteRecipes/Net_Remove_Unnecessary_StridedSlice_000/test.recipe

```diff
@@ -0,0 +1,92 @@
+operand {
+  name: "ifm"
+  type: INT32
+  shape { dim: 1 dim: 256 dim: 40 dim: 40 }
```

Comment:
Can you change to much smaller values so that interpreter execution time will be shorter?,

Reply:
Done